src/npcmove.cpp

```diff
@@ -669,24 +689,37 @@ void npc::assess_danger()
         assessment = std::max( min_danger, assessment - guy_threat * 0.5f );
     }
 
+    // being outnumbered is serious.  Do a flat scale up your assessment if you're outnumbered.
+    // This is a coarse tool that might be better handled
+    if( hostile_count > friendly_count + NPC_CROWD_BRAVADO ) {
+        assessment *= std::min( ( hostile_count / ( friendly_count + NPC_CROWD_BRAVADO ) ), 1.0f );
```

Comment:
The integer division here will chop off the decimal precision and round the ratio down, before it gets passed to the "assessment" float. If you want to preserve the precision, cast one of the components as a float before dividing.
```suggestion
        assessment *= std::min( ( hostile_count / static_cast<float>( friendly_count + NPC_CROWD_BRAVADO ) ), 1.0f );
```

Reply:
Woops, thanks. I knew about that one but I still miss them sometimes.